### DISCUSSION TITLE
Client agent restart , the server endpoint is not disappear

### DISCUSSION THREAD

**Discussion by shiyuanfunc** [apache/skywalking#7741] [Q&A] [answered]
version: 
oap: 6.6.0
ES: 7.0
agent client : 6.6.0 

![image](https://user-images.githubusercontent.com/19946732/133772986-a2082d8d-2f26-490b-883a-b87b8f4e4e64.png)

Actually, I only have two nodes，The previous node did not disappear

**Comment by wu-sheng** [comment, marked as answer]
Read time to live(TTL) doc.

**Reply by sdhzlzhk** [reply]
@songxiao-hui
 I also encountered the same problem, do you know how to solve it. I did not find any settings for the survival time of the agent instance in the official documentation

**Reply by hanahmily** [reply]
https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/ is what you need

**Reply by sdhzlzhk** [reply]
> Read time to live(TTL) doc.

Do you mean the "recordDataTTL"?

**Reply by sdhzlzhk** [reply]
> https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/ is what you need

Thank you. I got it.
